### DISCUSSION TITLE
Unpair while "Allow OB1 unbonding" disabled

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#1928] [Q&A]
Samsung A22

![271762472_3189065664660370_3119744111492135275_n](https://user-images.githubusercontent.com/51497406/149166714-24a4704c-d7a4-49db-9864-f8bef96d75a0.jpg)
![271763123_3189065761327027_5594403935179779592_n](https://user-images.githubusercontent.com/51497406/149166904-39f9e575-0fc0-464c-bc43-77d34caad11a.jpg)
![271731083_3188480001385603_448203908397369439_n](https://user-images.githubusercontent.com/51497406/149166946-564419b0-19e3-4b5f-9ead-df3d0cf4717c.jpg)
![271764561_3188882971345306_1966165288095714666_n](https://user-images.githubusercontent.com/51497406/149167014-014841c7-1783-42c1-8353-15d66da65fca.jpg)

These are from a post in facebook.
What I don't understand is why it still unbonds even though "Allow OB1 unbonding" is disabled.

**Comment by theunknown86** [comment]
This seems to be specific to the Pixel 6 from everything I've seen.

There seemed to be a fix where we turned off battery optimization on the Bluetooth system app which worked for a while, but I've started seeing it popup again the last month or so.

**Reply by Navid200** [reply]
The user finally responded to my question.  It's a Samsung A22.
